UI/locked-checkbox.cpp

```diff
@@ -7,10 +7,17 @@
 
 LockedCheckBox::LockedCheckBox() : QCheckBox()
 {
-	lockedImage =
-		QPixmap::fromImage(QImage(":/res/images/locked_mask.png"));
-	unlockedImage =
-		QPixmap::fromImage(QImage(":/res/images/unlocked_mask.png"));
+	if (devicePixelRatio() >= 2) {
+		lockedImage =
+			QPixmap::fromImage(QImage(":/res/images/locked_mask@2x.png"));
```

Comment:
Style:  These lines are longer than 80-characters.  You will probably have to line break them differently, like so:
```
		lockedImage = QPixmap::fromImage(QImage(
				":/res/images/locked_mask@2x.png"));
```

Reply:
Will fix